packages/create-sitecore-jss/src/common/processes/transform.ts

```diff
@@ -290,6 +291,12 @@ export const transform = async (
         // merge them and set the result to str which will then go through diff
         const merged = merge(currentPkg, templatePkg);
         str = JSON.stringify(merged, null, 2);
+      } else if (file.endsWith('.json') && fs.existsSync(pathToNewFile)) {
```

Comment:
See comment in helpers. Seems like we could make this generic and just have single case which handles all .json files.